crates/swc_ecma_lexer/src/common/lexer/mod.rs

```diff
@@ -1112,26 +1112,31 @@ pub trait Lexer<'a, TokenAndSpan>: Tokens<TokenAndSpan> + Sized {
         op(self, &mut buf)
     }
 
-    fn read_unicode_escape(&mut self) -> LexResult<Vec<Char>> {
```

Comment:
cc @bvanjoi Do you think this change is correct?

Reply:
If I understand correctly, this implementation appears incorrect. For `read_unicode_escape('\u0041')`, we expect the raw character sequence `['\\', 'u', '0', '0', '4', '1']` as output, not the decoded `'A'` directly.